source/_integrations/synologydsm.markdown

```diff
@@ -50,6 +50,11 @@ password:
   description: The password of the user to connect to the Synology NAS.
   required: true
   type: string
+api_version:
+  description: Define DSM version to allow backward compatibility with 5.x
+  default: 6
```

Comment:
According to the code (Voluptuous schema), there is no default?

Reply:
Oh yes, sorry, I mixed things up... There is no default here